### DISCUSSION TITLE
Localization fallbacks

### DISCUSSION THREAD

**Discussion by mattbrailsford** [umbraco/Umbraco-CMS#10707] [Features and ideas]
Currently, if a localized string can't be localized, it is displayed as `[alias]` which I don't think is that friendly. 

Maybe we could implement localization fallbacks / finders to allow someone to return an alternative value if something can't be found in a localization file.

**Comment by nul800sebastiaan** [comment]
I believe it will always show English if English is available, I believe it would be the US-English one that needs to be there. If en-US is not available you will see `[alias]`.
